Apps/WebClient/src/Server/Startup.cs

```diff
@@ -145,10 +146,9 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseSpa(spa =>
             {
                 spa.Options.SourcePath = "dist";
-
                 if (env.IsDevelopment())
-                {
-                    spa.UseProxyToSpaDevelopmentServer("http://localhost:5000");
+                {
+                    spa.UseReactDevelopmentServer("dev");

```

Comment:
Is this tied to the package.json line "dev"?

Reply:
Yep!